sql_export/security/sql_export_security.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-<data noupdate="0">
+<odoo noupdate="0">
```

Comment:
`noupdate="0"` is the default, you can remove that, or use `noupdate="1"`, which is probably what the author wanted in the first place.